### PR TITLE
[FIX] event: google map on event template

### PR DESCRIPTION
**Pull request by jpr-odoo**
Description of the issue/feature this PR addresses:
TaskID: https://www.odoo.com/web?#id=2069312&action=327&model=project.task&view_type=form&menu_id=4720
Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by chienandalu — COMMENTED** [review]
When website_event is not installed, won't the gmap image still be failing?

**Comment by sbuhl** [comment]
@jpr-odoo Please, review your commit message:
https://www.odoo.com/documentation/13.0/reference/guidelines.html#commit-message-structure

File: addons/event/data/email_template_data.xml [hunk 1]
```diff
@@ -169,8 +169,12 @@
     </table>
     % if object.event_id.address_id:
     <div style="width:598px;margin:0px auto;border-left:1px solid #dddddd;border-right:1px solid #dddddd;border-bottom:1px solid #dddddd;">
-        <a href="https://maps.google.com/maps?q=${location}" target="new">
-            <img src="http://maps.googleapis.com/maps/api/staticmap?autoscale=1&size=598x200&maptype=roadmap&format=png&visual_refresh=true&markers=size:mid%7Ccolor:0xa5117d%7Clabel:%7C${location}" style="vertical-align:bottom;" />
+        <a href="${object.event_id.google_map_link()}" target="new" style="text-decoration:none;color:#875A7B;">
```
Comment (mart-e) [inline addons/event/data/email_template_data.xml:172]: if you are going to break the translations in stable version, please update the .pot files

Comment (jpr-odoo) [reply addons/event/data/email_template_data.xml:172]: > if you are going to break the translations in stable version, please update the .pot files

I have updated the .pot files, please have a look

Comment (mart-e) [reply addons/event/data/email_template_data.xml:172]: Thanks !

**Comment by JKE-be** [comment]
v11 is no more supported.

To check if still needed and reopen new PR right version if needed.

Thanks

**Pull request closed without merging**